app/workers/annual_gift_removal_worker.rb

```diff
@@ -0,0 +1,9 @@
+require 'sidekiq-scheduler'
+
+class AnnualGiftRemovalWorker
+  include Sidekiq::Worker
```

Comment:
You can add a `BaseWorker` class which will be inherited by every worker. In this class, you can `include Sidekiq::Worker` and define some default options such as retries, and override them in every specific worker if needed